docs/products/clickhouse/reference/limitations.md

```diff
@@ -165,12 +162,12 @@ on how to work around specific restrictions.
 
 Service limits are determined by a plan that this service uses.
 
-| Plan     | VMs    | CPU per VM         | RAM per VM | Total storage   |
-| -------- | ------ | ------------------ | ---------- | --------------- |
-| Hobbyist | 1      | 1 (2 for AWS only) | 4 GB       | 180 GB          |
-| Startup  | 1      | 2                  | 16 GB      | 1150 GB         |
-| Business | 3      | 2 - 8              | 16 - 64 GB | 1150 - 4600 GB  |
-| Premium  | 6 - 30 | 2 - 8              | 16 - 64 GB | 2300 - 46000 GB |
+| Plan     | VMs    | CPU per VM         | RAM per VM | Total storage   | Maximum concurrent queries                                                                       |
+| -------- | ------ | ------------------ | ---------- | --------------- | ------------------------------------------------------------------------------------------------ |
+| Hobbyist | 1      | 1 (2 for AWS only) | 4 GB       | 180 GB          | 25 queries per 4 GB of RAM                                                                       |
+| Startup  | 1      | 2                  | 16 GB      | 1150 GB         | 100 queries per 16 GB of RAM                                                                     |
+| Business | 3      | 2 - 8              | 16 - 64 GB | 1150 - 4600 GB  | 100 queries per 16 GB of RAM<br/> 200 queries per 32 GB of RAM<br/> 300 queries per 64 GB of RAM |
```

Comment:
Sorry I actually made a mistake in the ticket: for 64GB of RAM, it is actually 400 max concurrent queries my bad! (16x4 = 64; 100x400 = 400)

Reply:
Since we just have multiple of 16 available for RAM actually, now that I see it in table format, should we just leave for Business and Premium "100 queries per 16 GB of RAM" - meaning if you have 2 times more you should multiply? Or should we do the calculation for them? Your call!